emacs/init.sls

```diff
@@ -1,2 +1,108 @@
-emacs:
-  pkg.installed
+{% from 'emacs/settings.sls' import emacs with context %}
+
+{%- if emacs.from_pkg == True %}
+## This is the default behavior, milk-toast if you will.
+boring-emacs:
```

Comment:
Could you please rename this to something a little more professional and remove the joke comments on lines 4 and 10?